### PR TITLE
Luca 2

### PR DESCRIPTION
**Pull request by LucRome**
rules by Felix

### CODE REVIEW

File: Projekt/PokerLibrary/CardAndSymbols.h [hunk 1]
```diff
@@ -16,6 +16,9 @@ enum values {
 struct card { //suits, value
 	suits suit = s_none;
 	values value = v_none;
+	const bool operator== (const card& c2) {
+		return this->suit == c2.suit && this->value == c2.value;
+	}
```
Comment (LucRome) [inline Projekt/PokerLibrary/CardAndSymbols.h]: for Tests (Expect_EQ(card, card))

File: Projekt/PokerLibrary/CardAndSymbols.h [hunk 2]
```diff
@@ -16,6 +16,9 @@ enum values {
 struct card { //suits, value
 	suits suit = s_none;
 	values value = v_none;
+	bool operator== (const card& c2) const {
+		return this->suit == c2.suit && this->value == c2.value;
+	}
```
Comment (LucRome) [inline Projekt/PokerLibrary/CardAndSymbols.h:21]: can be used for tests (EXPECT_EQ)

**Review by haasmi — COMMENTED** [review]
Looks good gonna start to test right away

**Comment by haasmi** [comment]
Looks good gonna start to test right away

**Review by haasmi — APPROVED** [review]

**Pull request merged**